recipe/recipe.yaml

```diff
@@ -46,7 +46,9 @@ source:
 
 build:
   number: ${{ build_number }}
-  skip: win
+  skip:
+    - win
+    - osx-64
```

Comment:
This skip doesn't exist. Although the inner workings of yaml are inscrutable to me, what I think is happening is that you end up with `bool({true|false}-64)`, wich is always True

Reply:
`osx and x86_64`